Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -469,6 +469,8 @@ void CGame::DoPulse(void)
 
     CLOCK_CALL1(m_pAsyncTaskScheduler->CollectResults());
 
+    CLOCK_CALL1(m_pMapManager->GetWeather()->DoPulse(););

```

Comment:
`DoPulse(););` should that extra semicolon be there?

Reply:
It's in nearly every other line with CLOCK_CALL1, so wanted to "continue" this style o.o

Reply:
Oh I see — I was just confused because the one above doesn't use it! Should be fine.